src/main/webapp/i18n/de/result.json

```diff
@@ -60,7 +60,16 @@
                 "points": "Punkte",
                 "deductions": "Abzüge",
                 "tooltip": "Deine Punktabzüge werden begrenzt um den Einfluss von Folgefehlern durch Code-Issues abzuschwächen. Beachte, dass du gegebenenfalls mehrere Code-Issues lösen musst, bevor sich deine Punkte erhöhen."
-            }
+            },
+            "missing": {
+                "viewPrevious": "Letztes Ergebnis anzeigen",
+                "programmingFailedSubmission": {
+                    "message": "Kein entsprechendes Ergebnis verfügbar.",
+                    "tooltipOfflineIde": "Für deine letzte Einreichung konnte kein Ergebnis gefunden / nicht rechtzeitig generiert werden. Bitte überprüfe deinen Code manuell in deiner IDE, z.B. überprüfe, ob dein Code lokal kompiliert und korrekt funktioniert.",
+                    "tooltipOnlineIde": "Für deine letzte Einreichung konnte kein Ergebnis gefunden / nicht rechtzeitig generiert werden. Bitte überprüfe deinen Code soweit möglich manuell."
+                }
+            },
+            "notLatestSubmission": "<h6>Hinweis: Dieses Ergebnis und Feedback bezieht sich <strong>nicht</strong> auf die letzte Einreichung.</h6>"
```

Comment:
Please remove HTML tags from the translation. I'd make an exception for formatting tags (`<strong>` here) if they are used for such short words. => move the h6 to the template